### ISSUE TITLE
validator warnings: Attribute “aria-activedescendant” value should either refer to a descendant element, or should be accompanied by attribute “aria-owns”.

### ISSUE THREAD

**Issue by nschonni** [w3c/aria-practices#1677]
Affected files:
```
"file:/mnt/z/aria-practices/examples/listbox/listbox-rearrangeable.html":75.13-75.117: info warning: Attribute “aria-activedescendant” value should either refer to a descendant element, or should be accompanied by attribute “aria-owns”.
"file:/mnt/z/aria-practices/examples/listbox/listbox-rearrangeable.html":135.13-141.42: info warning: Attribute “aria-activedescendant” value should either refer to a descendant element, or should be accompanied by attribute “aria-owns”.
```

**Comment by mcking65** [comment]
Warning needs investigation for validity; I thought we raised a validator issue because null or empty value should be allowed for activedescendant.

**Comment by nschonni** [comment]
https://github.com/w3c/aria-practices/issues/429 has the tracking for a similar issue that is suppressed right now with `aria-activedescendant` and an upstream issue
